Cognite.Simulator.Utils/ScopedRemoteApiSink.cs

```diff
@@ -96,11 +96,13 @@ private async Task SendToRemoteApi(SimulatorsResource client, CancellationToken
                     // to make sure we remove only the logs that were sent to the remote API
                     if (logBuffer.TryRemove(log.Key, out var logData))
                     {
-                        await client.UpdateLogsBatch(
-                            log.Key,
-                            logData,
-                            token
-                        ).ConfigureAwait(false);
+                        if(log.Key != 0){
+                            await client.UpdateLogsBatch(
+                                log.Key,
+                                logData,
+                                token
+                            ).ConfigureAwait(false);   
+                        }
```

Comment:
can we please not assign 0 and make a value nullable or smth?
defaulting things to 0 make these sort of bugs to happen in the first place

Reply:
the defaulting logic happens somewhere in the library as well I assume